src/mbox/message-box.lisp

```diff
@@ -179,6 +198,23 @@ This function sets the result as `handler-result' in `item'. The return of this
               (bt2:condition-notify withreply-cvar)))
           (handler-fun)))))
 
+
+(declaim (ftype (function (message-box/bt)
```

Comment:
What's the purpose of the declaim?

Reply:
Recently I've discovered that such type information sometimes helps to find issues during the compilation. So I started to add declarations to the code I'm touching in my own projects and also decided it would not harm to use here.

But if you wish, I'll remove it.